loader/src/sources/albertsons/index.js

```diff
@@ -11,8 +11,13 @@
  */
 
 const Sentry = require("@sentry/node");
+const _ = require("lodash");
```

Comment:
I think the bundle size may be smaller if you just `const groupBy = require("lodash/groupBy")`.

Reply:
I already started using `mapKeys` in #474, so I think it’s probably easier to pull in all of lodash as a dependency so expanding usage isn’t a pain (and we aren’t worried about running in a browser here). We’ll see if I regret that later, though.